src/main/java/attendance/domain/AttendancePolicy.java

```diff
@@ -0,0 +1,61 @@
+package attendance.domain;
+
+import static attendance.domain.AttendanceType.ABSENCE;
+import static attendance.domain.AttendanceType.ATTENDANCE;
+import static attendance.domain.AttendanceType.LATE;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class AttendancePolicy {
+    private static final LocalTime OPEN_TIME = LocalTime.of(8, 0);
+    private static final LocalTime CLOSE_TIME = LocalTime.of(23, 0);
+
+    private AttendancePolicy() {}
+
+    public static AttendanceType checkAttendanceType(LocalDate localDate, LocalTime localTime) {
+        if (localTime.isBefore(OPEN_TIME) || localTime.isAfter(CLOSE_TIME)) {
+            throw new IllegalArgumentException("캠퍼스 운영 시간이 아닙니다.");
+        }
+        DayOfWeek dayOfWeek = localDate.getDayOfWeek();
+        if (dayOfWeek == DayOfWeek.MONDAY) {
+            return checkMondayAttendanceType(localTime);
+        }
+        return checkGeneralAttendanceType(localTime);
+    }
+
+    public static void checkHoliday(LocalDate localDate) {
+        DayOfWeek dayOfWeek = localDate.getDayOfWeek();
+        if (dayOfWeek != DayOfWeek.SATURDAY && dayOfWeek != DayOfWeek.SUNDAY &&
```

Comment:
```suggestion
        if (isNotSaturday(dayOfWeek) && isNotSunday(dayOfWeek) && isNotChristmas(localDate)) {
```
적절한 메서드로 감싸면 좀 더 코드 파악하기에 좋죠

Reply:
해당 메서드에서는 주말 혹은 공휴일인지 체크하는 것이기 때문에 isWeekend와 isHoliday로 메서드를 추출했습니다.